app/api/chat/route.ts

```diff
@@ -17,20 +16,35 @@ export async function POST(request: Request) {
       .parse(body);
 
     const db = getDb();
+    const isPostgres = !!process.env.POSTGRES_URL;
+    
+    let ChatSessionsTable, InboxesTable, ContactsTable;
+    
+    if (isPostgres) {
+      const schema = await import("@/db/schema");
+      ChatSessionsTable = schema.ChatSessionsTable;
+      InboxesTable = schema.InboxesTable;
+      ContactsTable = schema.ContactsTable;
+    } else {
+      const schema = await import("@/db/sqlite-schema");
+      ChatSessionsTable = schema.ChatSessionsTable;
+      InboxesTable = schema.InboxesTable;
+      ContactsTable = schema.ContactsTable;
+    }
```

Comment:
We should not create a new schema. Any complexity about what db we're interacting with should be in the getDB method